src/main/java/io/sudhakar/student/dto/EmployeeGetAll.java

```diff
@@ -0,0 +1,16 @@
+package io.sudhakar.student.dto;
+
+import lombok.*;
+
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Data
+public class EmployeeGetAll {
```

Comment:
changeClass Name as EmployeeResponseGetAll

Reply:
changed class name from EmployeeGetAll to EmployeeResponseGetAll